include/seahorn/Expr/EvalModel.hh

```diff
@@ -0,0 +1,198 @@
+#pragma once
+
+#include "seahorn/Expr/EvalUtils.hh"
+#include "seahorn/Expr/ExprCore.hh"
+#include "seahorn/Expr/ExprGmp.hh"
+#include "seahorn/Expr/ExprOpBind.hh"
+#include "seahorn/Expr/ExprOpBinder.hh"
+
+#include <math.h>
+#include <stdlib.h>
+
+namespace expr {
+namespace eval {
+
+template <typename T> struct BvNum {
```

Comment:
add comment with intended meaning

Reply:
shouldn't width be not always stored as part of `BvNum`? Some representation of `T` might have width already stored in them